File: 1281/solution.c
int subtractProductAndSum(int n) {
    int product = 1;
    int sum = 0;
    char str[20];
    sprintf(str,"%d", n);	
    for (int i = 0; i < strlen(str); i++) {
        int x = str[i] - '0';
        product *= x;
        sum += x;
    }
    return product - sum;
}

File: 1281/solution.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function(n) {
    let str = String(n)
    let product = 1
    let sum = 0

    for(let i = 0; i < str.length; i++){
        x = Number(str[i])
        product *= x
        sum += x
    }

    return product - sum
};
